content/pages/pricing.html

```diff
@@ -160,31 +183,31 @@ <h3 class="ui small header">
       </div>
 
       <div class="ui basic center aligned padded segment">
-        <a class="ui teal button" href="https://readthedocs.org/accounts/signup/" data-analytics="community-signup">Sign up for a Community account</a>
+        <a class="ui teal button" href="https://readthedocs.org/accounts/signup/" data-analytics="community-signup">Sign up for a free Community account</a>
```

Comment:
```suggestion
        <a class="ui teal button" href="https://readthedocs.org/accounts/signup/" data-analytics="community-signup">Sign up for a free community account</a>
```

I see that Anthony always says that there is no need to use uppercase in these cases. So, I'm suggesting the same 😄